### ISSUE TITLE
How to test using mx.mod.Module?

### ISSUE THREAD

**Issue by ilkarman** [apache/incubator-mxnet#3158]
I have the following code to test my model:

```
    metric = mx.metric.Accuracy()
    for batch in load_data_frame(X_data=X_test,
                                 y_data=y_test,
                                 batch_size=BATCH_SIZE):
        mod.forward(batch, is_train=False)
        mod.update_metric(metric, batch.label)

    metric_m, metric_v = metric.get()
    print("TEST(%s): %.4f" % (metric_m, metric_v))
```

However, it takes as long to test as to train (even a bit longer) and I'm worried I'm training it by mistake or doing something way wrong.

**Comment by pluskid** [comment]
This looks correct. Is your test set bigger than training set?

**Comment by phunterlau** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!
